src/utils.ts

```diff
@@ -102,14 +102,14 @@ export namespace Utils {
    * @param {string} challengeTx SEP0010 challenge transaction in base64.
    * @param {string} serverAccountID The server's stellar account (public key).
    * @param {string} networkPassphrase The network passphrase, e.g.: 'Test SDF Network ; September 2015'.
-   * @param {string} [homeDomain=undefined] The field is reserved for future use and not used.
-   * @returns {Transaction|string} The actual submited transaction and the stellar public key (master key) used to sign the Manage Data operation.
+   * @param {string} [homeDomain=undefined] The home domain that should be included in the Manage Data operation's string key.
+   * @returns {Transaction|string} The actual submitted transaction and the stellar public key (master key) used to sign the Manage Data operation.
```

Comment:
💡 I think we should probably avoid the word submitted here because submitting is the term usually used in our APIs for when the transaction is sent to Horizon and on to Core. I'm not opposed to it though if this makes this easier to understand.
```suggestion
   * @returns {Transaction|string} The actual transaction and the stellar public key (master key) used to sign the Manage Data operation.
```